notebooks/00_quick_start/sar_movielens_with_azureml.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p>Are we sure that we want to explicitly set the current price in the text? this could change without notice</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
We've gotten feedback that users want to see how much pricing costs. We could just link to the pricing calculator, but I think it's good to explicitly call it out. We should make it more clear that it is an estimated amount so we are not bound to that price @jingyanwangms

Reply:
I've seen in other examples that we link to the pricing calculator like in [this one](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/ai/batch-scoring-databricks#cost-considerations), in this other [example](https://docs.microsoft.com/en-us/azure/architecture/example-scenario/ai/intelligent-apps-image-processing#pricing), there is a link to a preconfigured estimation using the calculator. Maybe this last is a good solution, it would provide an exact price but it will be updated dynamically, so we don't have to add a number that will change for sure

Reply:
I posted our pricing discussion in teams. You're right, we decided not to spell the exact dollar amount. I'd personally prefer exact amount with a disclaimer just because $1 and $9 is very different. But vague words is the option withlowest maintenance.